lms/djangoapps/badges/migrations/0002_data__migrate_assertions.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import json
+
+import os
+from django.db import migrations, models
+
+
+def forwards(apps, schema_editor):
+    """
+    Migrate the initial badge classes, assertions, and course image configurations from certificates.
```

Comment:
@nedbat It depends on the configuration where the files are actually stored. I don't imagine these tables will be /too/ massive. The only existing badges are those awarded at the end of a course, and this functionality has been around less than a year. So you'd figure however many students enrolled in badge-aware courses passed them during that time period for the assertions, and however many badge-aware courses there were for the badge classes.

The course image configurations will always be a small number, 3 or 4 or so.